README.md

```diff
@@ -1,2 +1,62 @@
-# youtube-project
-Project: Embedding Images and Video
+# Youtube Clone Page
+
+> This project consists of building an HTML document that matches the appearance of youtube video player page.
```

Comment:
**Change it to:** This project consists of building an HTML document that matches the appearance of the [youtube video player page](http://archive.vn/Bss88).